config/admin/layout/content--home-body.html

```diff
@@ -522,6 +522,12 @@ <h4 style="direction:rtl;">
       مرحبا بك في الدردشة الرسمية لFreecodecamp عرف نفسك هنا:
       <a href="/channel/Arabic">#Arabic</a>
     </h4>
+    
+    <h4 style="direction:rtl;">
+      <!--Urdu-->   
+      آفیشل FreeCodeCamp چیٹ سرود میں خوشامدید ! اپنا تعارف کروا ئیں!:
+      <a href="/channel/Urabic">#Urabic</a>
```

Comment:
```suggestion
      <a href="/channel/Urdu">#Urdu</a>
```
The channel was created as `#Urdu`. If it should be called `#Urabic`, let me know and I can rename the channel and dismiss this suggestion.

Reply:
It seems like i invented a language :)